uitools/examples/qml_quick/src/demos/BookmarksViewDemoForm.qml

```diff
@@ -0,0 +1,40 @@
+// Copyright 2022 ESRI
+//
+// All rights reserved under the copyright laws of the United States
+// and applicable international laws, treaties, and conventions.
+//
+// You may freely redistribute and use this sample code, with or
+// without modification, provided you include the original copyright
+// notice and use restrictions.
+//
+// See the Sample code usage restrictions document for further information.
+//
```

Comment:
This is entirely my fault. I think that even for the example app, that these headers are likely wrong. There is no sample code restrictions document to review in this repo. @ldanzinger, should the example app copyright headers be the Apache license ones?

Reply:
I do not know what copyright headers are required - I defer to the judgement of @ldanzinger

Reply:
Everything in this repo should be Apache license. @mfeigl can you open up a new issue and we can address it there please?